Makefile

```diff
@@ -1,8 +1,32 @@
+.PHONY: clean clean_all
+
 PROJ_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
-# Configuration of extension
-EXT_NAME=inet
-EXT_CONFIG=${PROJ_DIR}extension_config.cmake
+# Main extension configuration
+EXTENSION_NAME=inet
+
+# Set to 1 to enable Unstable API (binaries will only work on TARGET_DUCKDB_VERSION, forwards compatibility will be broken)
+# WARNING: When set to 1, the duckdb_extension.h from the TARGET_DUCKDB_VERSION must be used, using any other version of
+#          the header is unsafe.
+USE_UNSTABLE_C_API=1
```

Comment:
Does it make sense to open an issue that lists all the C API functions that we need to stabilize to be able to switch inet to the stable C API?